packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -436,6 +436,14 @@ This function allows you to use your own `jsx` template for the items in the dro
 
 The variant of the Combobox. Available variants are `default` or `inverse`.
 
+##### `getA11yStatusMessage?: () => void`
+
+Useful to compose accessible status messages to assistive technology users, including translations.
+
+##### `getA11ySelectionMessage?: () => void`
+
+Useful to compose accessible status messages to assistive technology users, including translations.
```

Comment:
Should this say "selection" instead of "status"? It's the exact same description as the line above.

Reply:
It should, yes!!